ui/apps/platform/src/Containers/MainPage/Sidebar/NetworkGraphNavItems.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { useLocation, Location } from 'react-router-dom';
+import { networkBasePath, networkBasePathPF } from 'routePaths';
+import { Badge, Flex, FlexItem } from '@patternfly/react-core';
+
+import { IsFeatureFlagEnabled } from 'hooks/useFeatureFlags';
+
+import LeftNavItem from './LeftNavItem';
+
+type NetworkGraphNavItemsProps = {
+    isFeatureFlagEnabled: IsFeatureFlagEnabled;
+};
+
+function NetworkGraphNavItems({ isFeatureFlagEnabled }: NetworkGraphNavItemsProps) {
+    const location: Location = useLocation();
+    const isNetworkGraphPFEnabled = isFeatureFlagEnabled('ROX_NETWORK_GRAPH_PATTERNFLY');
+
+    const networkGraphTitle = isNetworkGraphPFEnabled ? (
+        <Flex>
+            <FlexItem>Network Graph</FlexItem>
+            <FlexItem>
+                <Badge
+                    style={{
+                        backgroundColor: 'var(--pf-global--palette--cyan-400)',
+                    }}
+                >
+                    1.0 deprecated
+                </Badge>
```

Comment:
Considering the page has a warning alert, how about plain text instead of badge so the left nav aligns:
* Network Graph (2.0)
* Network Graph (1.0 deprecated)

Although the special style annoyed my eyes for **preview** at least it attracted them **to the new** graph.

Now it pulls my eyes **toward the old** graph and pushes them **away from the new** graph.

Reply:
@zhenpesky @boazmichaely I think Mark has a good point here. After looking at it again, it pulls your attention toward the old graph. If it is good with both of you, I think Mark's suggestion might be a good alternative

Reply:
Another alternative might be to actually add a badge for the new network graph to pull the attention back to that